src/main/resources/static/homepage.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <title>Airports - Homepage</title>
+</head>
+<body>
+<br><br><br><br><br>
+<center>
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+        <style>
```

Comment:
I think it's bad practice to pollute the markup with CSS code when you can place it in a dedicated file.

Reply:
moved CSS code into a CSS file